### PR TITLE
PR changes

### PR DESCRIPTION
**Pull request by steveburton4**

### CODE REVIEW

File: hsm-client/hsm-client_install.sh [hunk 1]
```diff
@@ -17,12 +17,10 @@ sudo pip install awscli
...
 # Install HSM client, the SDK and dependencies
 sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-latest.el7.x86_64.rpm
-sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-pkcs11-latest.el7.x86_64.rpm
-sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-dyn-latest.el7.x86_64.rpm
 sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-jce-latest.el7.x86_64.rpm
 
-sudo mkdir /opt/hsm-client
-sudo mkdir /var/log/hsm-client
+# Disable the service from starting at startup
+service cloudhsm-client disable
```
Comment (benjaminsherwood) [inline hsm-client/hsm-client_install.sh]: **systemctl disable cloudhsm-client**

CentOS is sysd, we're striving for consistency.

Comment (steveburton4) [reply hsm-client/hsm-client_install.sh]: I did wonder but @mattburgess was very specific 😆

Comment (mattburgess) [reply hsm-client/hsm-client_install.sh]: Aye, but the docs say `service` so...:shrug

Comment (steveburton4) [reply hsm-client/hsm-client_install.sh]: Changed

**Pull request merged**